models/migrations/v1_21/v276.go

```diff
@@ -63,6 +64,8 @@ func migratePullMirrors(x *xorm.Engine) error {
 		var mirrors []Mirror
 		if err := sess.Select("mirror.id, mirror.repo_id, mirror.remote_address, repository.owner_name as repo_owner, repository.name as repo_name").
 			Join("INNER", "repository", "repository.id = mirror.repo_id").
+			Where("repository.status = ?", repo_model.RepositoryReady).
```

Comment:
This is for migrations, it may/may not have the mirrors. So we cannot assume it always have when the status = RepositoryPendingTransfer. We can only assume when status = RepositoryPendingReady, the git data have the remote.